### PR TITLE
feature: Truncate branch names to avoid breaking k8s deployments :breaking:

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

**Comment by machadoit** [comment]
I like! Would add breaking tag though

**Review by bmbferreira — CHANGES_REQUESTED** [review]
Hey folks! Can this be optional/parametrized?

**Comment by lolgab** [comment]
> Hey folks! Can this be optional/parametrized?

Do you have a use case that requires the non-truncated normalized branch name in the tag?

File: src/main.cr [hunk 1]
```diff
@@ -14,7 +14,7 @@ log_paths = ""
 
 folder = FileUtils.pwd
 
-OptionParser.parse! do |parser|
+OptionParser.parse do |parser|
```
[the remark this answers is not in the document]
Comment (lolgab) [reply src/main.cr:17]: It was failing to compile since they changed it in Crystal 1

**Review by machadoit — APPROVED** [review]
Most of the simplifications seem great assuming that things still work! Good job!

**Comment by lolgab** [comment]
@bmbferreira We're merging this as is. You can use the previous version if you really need this behavior, since the two versions are completely the same, except for this change. Let us know your use case if you really want the flag :)

**Pull request merged**